app/ui/src/app/integrations/edit-page/step-configure/filter-steps/basic-filter.component.ts

```diff
@@ -115,8 +115,7 @@ export class BasicFilterComponent implements OnChanges {
     this.integrationSupport
       .getFilterOptions(dataShape)
       .toPromise()
-      .then((resp: any) => {
-        const body = JSON.parse(resp['_body']);
+      .then((body: any) => {
```

Comment:
You don't want this change on this branch, this version still uses the restangular impl.

Reply:
If this is fixed, I'm happy to merge (and kick of a new TP3 release)